doc/manual.lyx

```diff
@@ -424,7 +424,7 @@ $NRUNS$
 \end_inset
 
```

Comment:
I think you need to merge `master` into branch and then reapply your changes - I see some parts are being removed with merge here